package.json

```diff
@@ -7,6 +7,7 @@
     "packages/*"
   ],
   "devDependencies": {
+    "@playwright/test": "^1.27.1",
```

Comment:
@freuleinkorksi can you remove it? this shouldn't be here and is not needed to run the test

Reply:
will do